knimin/tests/test_ag_pulldown.py

```diff
@@ -53,6 +55,13 @@ def test_get(self):
             self.assertIn("<option value='%s'>%s</option>" % (survey, survey),
                           response.body)
         self.assertNotIn('<input type="submit" disabled>', response.body)
+        for (id, name, selected) in db.list_ag_surveys():
```

Comment:
remember the definition of list_ag_surveys. There it is said that all get "selected" if selected is not set, i.e it is None. So in fact, all are selected in this case.